drgn/helpers/linux/tcp_states.py

```diff
@@ -0,0 +1,54 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+"""
+TCP States
+----------
+
+The ``drgn.helpers.linux.tcp_states`` module provides definitions for the TCP
+protocol sk_state field defined in :linux:`include/net/tcp_states.h`.
+"""
+
+__all__ = [
+    'TcpStates'
+]
+
+
+class TcpStates(object):
```

Comment:
Python 3 has the [`enum`](https://docs.python.org/3/library/enum.html) module which we could use here.